app/models/project.js

```diff
@@ -18,5 +18,10 @@ export default Model.extend({
     }
 
     return this.get('terminal').promiseObjectProxy(promise);
-  }).readOnly()
+  }).readOnly(),
+
+  version: computed.alias('package.version'),
+  dependecies: computed('package.dependencies', 'package.devDependencies', function() {
+    return Ember.assign({}, this.get('package.dependencies'), this.get('package.devDependencies'));
```

Comment:
I think we should use here Ember.get in case when package will be null or there will be no dependencies specified